packages/rocketchat-api/server/routes.coffee

```diff
@@ -67,6 +67,39 @@ RocketChat.API.v1.addRoute 'chat.postMessage', authRequired: true,
 		catch e
 			return RocketChat.API.v1.failure e.error
 
+
+# Update Channel Message
+RocketChat.API.v1.addRoute 'chat.update', authRequired: true,
```

Comment:
Maybe we should name this a bit different? Saying chat update feels super generic for being a method to update a message.